### PR TITLE
support non traceable module in fine-grained FX

### PR DESCRIPTION
**Pull request by xin3he**
## Type of Change

support non traceable module in fine-grained FX

## Description

JIRA ticket: [ILITV-2652](https://jira.devtools.intel.com/browse/ILITV-2652)

## Expected Behavior & Potential Risk

UT passes and coverage doesn't reduce.

### CODE REVIEW

File: neural_compressor/adaptor/pytorch.py [hunk 1]
```diff
@@ -2997,22 +2997,27 @@ def quantize(self, tune_cfg, model, dataloader, q_func=None):
                     q_model._model = prepare_qat_fx(
                         q_model._model,
                         self.fx_op_cfgs,
-                        example_inputs=self.example_inputs,
-                        prepare_custom_config=self.prepare_custom_config_dict)
+                        example_inputs=example_inputs,
```
Comment (PenghuiCheng) [inline neural_compressor/adaptor/pytorch.py:3000]: where example_inputs is defined?

File: neural_compressor/adaptor/pytorch.py [hunk 2]
```diff
@@ -2997,22 +2997,27 @@ def quantize(self, tune_cfg, model, dataloader, q_func=None):
...
-                                                    prefix='',
-                                                    is_qat=True,
-                                                    example_inputs=self.example_inputs)
+                PyTorch_FXAdaptor.prepare_sub_graph(
+                    self.sub_module_list,
+                    self.fx_op_cfgs,
+                    q_model._model,
+                    prefix='',
+                    is_qat=True,
+                    example_inputs=example_inputs,
```
Comment (PenghuiCheng) [inline neural_compressor/adaptor/pytorch.py:3018]: same with above comments.

File: neural_compressor/adaptor/pytorch.py (hunk 1 above)
Comment (xin3he) [reply neural_compressor/adaptor/pytorch.py:3000]: sorry， mistake

File: neural_compressor/adaptor/pytorch.py [hunk 3]
```diff
@@ -3030,32 +3035,39 @@ def quantize(self, tune_cfg, model, dataloader, q_func=None):
...
-                                                    self.fx_op_cfgs,
-                                                    q_model._model,
-                                                    prefix='',
-                                                    example_inputs=self.example_inputs)
+                PyTorch_FXAdaptor.prepare_sub_graph(
+                    self.sub_module_list,
+                    self.fx_op_cfgs,
+                    q_model._model,
+                    prefix='',
+                    example_inputs=example_inputs,
```
[the remark this answers is not in the document]
Comment (xin3he) [reply neural_compressor/adaptor/pytorch.py:3055]: sorry， mistake

**Review by PenghuiCheng — APPROVED** [review]

**Pull request merged**